### PR TITLE
Doc 297 logger

### PR DESCRIPTION
**Pull request by PaulineMauryL**

### CODE REVIEW

**Comment by PaulineMauryL** [comment]
## Coverage report


 

_This PR does not seem to contain any modification to coverable code._

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -90,3 +90,35 @@ The following actions must take place in this order when preparing a new release
...
+- `diffprivlib_logger` for diffprivlib pipelines: https://github.com/dscc-admin-ch/diffprivlib-logger
+- `smartnoise_synth_logger` for smartnoise_synth table transformer constraints: https://github.com/dscc-admin-ch/smartnoise-synth-logger
+
+Do not forget to add these libraries in the `requirements.txt` files.
+
+## Adding a Dataset Store
+TODO
+
+## Adding a Data Connector (for private dataset in various databases)
+TODO
```
Comment (PaulineMauryL) [inline CONTRIBUTING.md:124]: should be easy to do

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -90,3 +90,35 @@ The following actions must take place in this order when preparing a new release
 The workflows listed in the previous section will take care of building and publishing the different items (docker images, pip packages, etc.).
 
 Note: Helm charts are updated when there is a push on the `release/vx.y.z` branch. If you have a specific deployment that rely on the Chart, you can test it before finishing the release. Then, do not forget to update the chart and app versions of your specific deployment.
+
+## Adding a DP Library
+
+It is possible to add DP libraries quite seamlessly. Let0s say the new library is names 'NewLibrary'
```
Comment (damienbfs) [inline CONTRIBUTING.md:96]: Small typos: "let's say" and "is named"

**Review by damienbfs — APPROVED** [review]
Nice!

**Pull request merged**